test/hotspot/jtreg/vmTestbase/nsk/monitoring/ThreadMXBean/resetPeakThreadCount/reset001.java

```diff
@@ -21,6 +21,67 @@
  * questions.
  */
 
+/*
+ * @test
+ *
+ * @summary converted from VM Testbase nsk/monitoring/ThreadMXBean/resetPeakThreadCount/reset001..reset005
+ * VM Testbase keywords: [quick, monitoring]
+ * VM Testbase readme:
+ * DESCRIPTION
+ *     The test checks that
+ *         ThreadMXBean.resetPeakThreadCount()
+ *     returns correct result.
+ *     The test starts a couple of user threads and waits until they finish. After
+ *     that, resetPeakThreadCount() is invoked to reset the peak. Then
+ *     getPeakThreadCount() and getThreadCount() must return the same values. The
+ *     preposition is that no threads are appered/disappeared between
```

Comment:
This does not read correctly even ignoring the typo in "appered" - suggestion:

> The expectation is that no threads are created, or terminate, between ...

And please also fix the same comment at lines 118/119

Reply:
Done.
I'm not sure about commas (for me it's better without them)
Fixed here as you suggested and without commas in lines 118-119 :)
Please add a comment if you prefer consistency